A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_GPSMonitor.js

```diff
@@ -1,6 +1,11 @@
 class CDUGPSMonitor {
-    static ShowPage(mcdu) {
-        let currPos = new LatLong(SimVar.GetSimVarValue("GPS POSITION LAT", "degree latitude"), SimVar.GetSimVarValue("GPS POSITION LON", "degree longitude")).toShortDegreeString();
+    static ShowPage(mcdu, merit1, merit2, sat1, sat2) {
+        let currPos = new LatLong(SimVar.GetSimVarValue("GPS POSITION LAT", "degree latitude"), 
+                                  SimVar.GetSimVarValue("GPS POSITION LON", "degree longitude")).toShortDegreeString();
+        let currPosSplit = currPos.split("N")
```

Comment:
It is me or N is not a constant value?
It could be N or S depending on the position of the plane in the world.